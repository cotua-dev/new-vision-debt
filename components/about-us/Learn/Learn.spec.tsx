import { render, RenderResult } from '@testing-library/react';
import { Learn } from './index';

describe('Learn Component', (): void => {
    let renderResult: RenderResult;

    beforeAll((): RenderResult => renderResult = render(<Learn/>));

    test('renders', (): void => {
        const { container } = renderResult;
        expect(container.firstChild).toMatchSnapshot();
    });

    afterAll((): void => renderResult.unmount());
});
