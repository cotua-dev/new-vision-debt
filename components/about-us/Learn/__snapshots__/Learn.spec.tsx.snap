// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Learn Component renders 1`] = `
<section
  class="learn-about-us"
>
  <h2
    class="section-title"
  >
    Learn About Our Debt Program
  </h2>
  <details>
    <summary
      class="title"
    >
      What is debt restructure?
    </summary>
    <p
      class="content"
    >
      Debt restructure, also known as debt settlement, is a method of dealing with unsecured debt outside of bankruptcy. Clients, on a monthly basis, put money into savings. Once sufficient funds become available, one of our licensed partners will negotiate settlements of the client’s enrolled debt accounts with the client’s creditors
    </p>
  </details>
  <details>
    <summary
      class="title"
    >
      Can I enroll all of my debts into debt restructure program?
    </summary>
    <p
      class="content"
    >
      No, only unsecured debts are eligible for enrollment in a debt restructure program. Debt that is secured in some manner—whether by way of mortgage or security interest—does not qualify for a debt restructure program
    </p>
  </details>
  <details>
    <summary
      class="title"
    >
      Can anyone enroll into a debt restructure program?
    </summary>
    <p
      class="content"
    >
      No. Only qualified individuals are accepted into a debt restructure program. One important requirement is that an individual must have suffered from some form of financial hardship. The hardship must have materially affected the individual’s ability to repay debts when due
    </p>
  </details>
  <details>
    <summary
      class="title"
    >
      Why not do this myself?
    </summary>
    <p
      class="content"
    >
      While you certainly may perform similar services on your own behalf, our licensed partners are able to negotiate better settlements with creditors given their expertise and experience in the industry. Debt restructure can also be a very time-consuming process. Many of our clients appreciate the assistance and time savings that our licensed partners can provide
    </p>
  </details>
  <p
    class="section-content"
  >
    This company does not actually provide any debt settlement, debt consolidation, or other credit counseling services. We ONLY refer you to companies that want to provide some or all of those services
  </p>
</section>
`;
